lib/rules/function-url-quotes/index.js

```diff
@@ -58,22 +72,64 @@ const rule = (primary, secondaryOptions) => {
 		 * @param {import('postcss').AtRule} atRule
 		 */
 		function checkAtRuleParams(atRule) {
-			const atRuleParamsLowerCase = atRule.params.toLowerCase();
-
-			functionArgumentsSearch(atRuleParamsLowerCase, 'url', (args, index) => {
+			functionArgumentsSearch(atRule.params, /^url$/i, (args, index) => {
 				checkArgs(args, atRule, index + atRuleParamIndex(atRule), 'url');
 			});
-			functionArgumentsSearch(atRuleParamsLowerCase, 'url-prefix', (args, index) => {
+			functionArgumentsSearch(atRule.params, /^url-prefix$/i, (args, index) => {
 				checkArgs(args, atRule, index + atRuleParamIndex(atRule), 'url-prefix');
 			});
-			functionArgumentsSearch(atRuleParamsLowerCase, 'domain', (args, index) => {
+			functionArgumentsSearch(atRule.params, /^domain$/i, (args, index) => {
 				checkArgs(args, atRule, index + atRuleParamIndex(atRule), 'domain');
 			});
 		}
 
 		/**
+		 * @param {import('postcss').Declaration | import('postcss').AtRule} node
 		 * @param {string} args
-		 * @param {import('postcss').Node} node
+		 * @param {number} index
+		 */
+
+		function addQuotes(node, args, index) {
+			const fixedName = `"${args}"`;
+			const openIndex =
+				'params' in node ? index - atRuleParamIndex(node) : index - getDeclarationValueIndex(node);
```

Comment:
nit: I'm not sure if the following is clearer (or an alternative)
```suggestion
				index - ('params' in node ? atRuleParamIndex(node) : getDeclarationValueIndex(node));
```

Or, setting a variable to the value of `'params' in node ?  atRuleParamIndex(node) :  getDeclarationValueIndex(node)`, etc.

Reply:
Both seem acceptable, though I prefer the suggested code.